contrib/code/index.rst

```diff
@@ -9,7 +9,7 @@ Code contributions
    |purpose|
 
 * Code setup and building (more complex stuff, delta from basic setup above)
-* More git bootcamp (patches, etc)
+* More Git bootcamp (pull requests, etc)
```

Comment:
The Git bootcamp page is/should be a FAQ-like page with Git (and possibly GitHub CLI) commands, whereas the PR workflow is described in more detail in the lifecycle of a PR page (which is more howto-like).

Does the "more" here imply you are planning to expand this page to include other things?

Reply:
When I first wrote the outline, I thought the first Git bootcamp page would be simple things that applied to everyone, and the code section would have more Git stuff for the higher demands of coders.  Now I'm not so sure it will divide up like that.